application/service/address_test.go

```diff
@@ -48,15 +50,30 @@ func TestAddressService_Create(t *testing.T) {
 		{
 			Name: "success to save an address",
 			Address: domain.AddressDomain{
-				Street: "Rua Fulaninho da Silva",
-				Number: "123",
+				ID:           0,
+				Street:       "Rua Fulaninho da Silva",
+				Number:       "123",
+				Complement:   "303",
+				Block:        "A",
+				Neighborhood: "São Mateus",
+				ZipCode:      "36025-290",
+				City:         "Juiz de Fora",
+				State:        "MG",
```

Comment:
Achas válido criar constantes dos estados BR? Só perguntando, pois isso aumentaria a complexidade no momento.

Reply:
Elas seriam usadas só nos testes ne? Se for só pra isso, não sei se tem necessidade.

Reply:
Fiz uns ajustes doidos aqui pensando nisso. Como realmente era muita repetição dessa estrutura, criei uma função pra isso.